### ISSUE TITLE
oysttyer hangs when replying to deleted tweet

### ISSUE THREAD

**Issue by IvanSanchez** [oysttyer/oysttyer#69]
I've observed oysttyer hanging up a few times (no prompt, no new tweets, no error messages). I _think_ the conditions necessary to reproduce are:
- See a tweet from somebody appear on your timeline
- That somebody deletes the tweet (but you can still see it in oysttyer's UI)
- Reply to that with the usual `/re a0 whatever`
- See oysttyer hang up

I'm using d2d6af297 and perl 5.22.1, if that matters.

**Comment by Llammissar** [comment]
Is that repro reliable?  Because it sounds awfully similar to bug #8, which has been kind of a pain in the butt for me for a while now.  (Even though there was some conjecture that that one doesn't happen with 5.22.x, I'm not sure if that's true.)

**Comment by funnelfiasco** [comment]
Yeah, this sounds familiar. I most often notice it when replying (not to deleted tweets), but that's because it's more noticeable then.

**Comment by atomicules** [comment]
Wasn't really looking at this, but remembered this issue and just tested replying to myself on a deleted tweet and it just sent it through as a new tweet. 

I'll have to remember to test this when replying to someone else's deleted tweet.
